graders/python/python_autograder/pl_unit_test.py

```diff
@@ -26,6 +26,7 @@ class PLTestCase(unittest.TestCase):
     student_code_file = 'user_code.py'
     iter_num = 0
     total_iters = 1
+    ipynb_key = '#keep'
```

Comment:
```suggestion
    ipynb_key = '#grade'
```

I know this is bikeshedding, but would `#grade` make the purpose clearer here?

Reply:
Sure, we can do that :-)

My script currently change `#clear` to `#keep`, and that is an easy change!

Reply:
done!